core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -110,8 +115,9 @@ class SearchService(
       val outOfDate = fileCheck.map(_.changed).getOrElse(true)
       if (!outOfDate) Future.successful(None)
       else {
+        val boostFile = isUserFile(base.getName())
```

Comment:
(or maybe we should be passing that containing directory information along anyway, it would help to reduce `String` duplication and reduce the database size)